wasmtime-jni/src/io_github_kawamuray_wasmtime_Linker/imp.rs

```diff
@@ -66,6 +67,48 @@ impl<'a> JniLinker<'a> for JniLinkerImpl {
         Ok(ret)
     }
 
+    fn native_externs(
```

Comment:
I'd like to make APIs of this library a direct map of the original wasmtime APIs as possible as it can.
The API we're using here provides iterator over all items in linker along with its module name and name. So how about we follow that API and add a method `public Collection<ExternItem> externs()` with defining `ExternItem` to hold not only extern itself but also module name and name?

In addition to that, keeping these two methods you've added as a helper to access limited range of data (items under the particular module, and list of module names) is fine.

Reply:
Not 100% sure what you mean here. I have kept the native method the same and modified the Java API. I think that's what you meant - it was certainly more straightforward than changing the native method to return a new object type.